dataset.py

```diff
@@ -2,9 +2,9 @@
 import numpy as np
 import os.path
 
-import preprocessing
-import evaluation
-import models
+from preprocessing import *
```

Comment:
I personally prefer the previous way, but I'll leave this up to you. The reason I prefer `import preprocessing` is that now we are forced to call `preprocessing.my_preprocessing_fn()`, which makes it clear where this function is defined.